http4s/src/main/scala/com/avast/grpc/jsonbrige/http4s/Http4s.scala

```diff
@@ -0,0 +1,67 @@
+package com.avast.grpc.jsonbrige.http4s
+
+import cats.data.NonEmptyList
+import cats.effect.IO
+import com.avast.grpc.jsonbridge.GrpcJsonBridge
+import io.grpc.Status.Code
+import io.grpc.{BindableService, Status => GrpcStatus}
+import org.http4s.dsl.impl.Root
+import org.http4s.dsl.io._
+import org.http4s.headers.`WWW-Authenticate`
+import org.http4s.{Challenge, HttpService, Response}
+
+import scala.concurrent.ExecutionContext
+
+object Http4s {
+  def apply(configuration: Configuration)(bridges: GrpcJsonBridge[_ <: BindableService]*)(
+      implicit ec: ExecutionContext): HttpService[IO] = {
+    val services = bridges.map(s => (s.serviceName, s): (String, GrpcJsonBridge[_])).toMap
+
+    HttpService[IO] {
+      case request @ POST -> Root / serviceName / methodName =>
```

Comment:
I imagine I could want the "base path" configurable, especially when bridging gRPC calls alongside having other app-specific routes. Maybe adding a parameter to the config, with `Root` being the default?